pylops/transforms/discreetcosine.py

```diff
@@ -0,0 +1,71 @@
+__all__ = ["DiscreetCosine"]
```

Comment:
Note that you mispelled this also in the PR title, it is discrete, not discreet.

Also, once again to be consistent with the current codebase conventions, I would call this operator DCT (since we have FFT and DWT…)

Reply:
updated in [af77480](https://github.com/PyLops/pylops/pull/480/commits/af77480d4d0a33b1b27afcc28e70fcefdbc0d6dd)

Reply:
fixed in [289d51a](https://github.com/PyLops/pylops/pull/480/commits/289d51ad61c7294451d777af6367ea282ad1e78e)